packages/react/src/components/Switch/Switch.js

```diff
@@ -32,8 +39,11 @@ const Switch = props => {
   const handleKeyDown = e => {
     const key = e.key || e.which;
 
-    if (key === 'Enter' || key === 13 || key === ' ' || key === 32) {
-      onKeyDown({ index, name, text });
+    if (match(key, 'ArrowRight')) {
```

Comment:
Should use the exported `keyCodes` to keep things consistent:

```suggestion
    if (match(key, keyCodes.RIGHT)) {
```